pkg/apis/syn/v1alpha1/tenant_types.go

```diff
@@ -10,6 +10,12 @@ type TenantSpec struct {
 	DisplayName string `json:"displayName,omitempty"`
 	// GitRepoURL git repository storing the tenant configuration. If this is set, no gitRepoTemplate is needed.
 	GitRepoURL string `json:"gitRepoURL,omitempty"`
+	// GitRepoRev allows to configure the revision of the tenant configuration to use. It can be any git tree-ish reference. Defaults to HEAD if left empty.
+	GitRepoRev string `json:"gitRepoRev,omitempty"`
```

Comment:
Standardize on `Rev` or `Revision`, but don't use one variant here and the other for the cluster CR.

Reply:
Seems like I got lost somewhere in my decision making. That was certainly not my intention.

Reply:
I'd rather go with `GitRepoRevision` since it's a lot clearer IMHO.

Reply:
I took inspiration from `TenantRef`. But here you go, changed.